content/en/continuous_integration/setup_tests/swift.md

```diff
@@ -29,8 +29,12 @@ There are two ways of installing the testing framework:
 
 1. Add `dd-sdk-swift-testing` package to your project. It is located at [`https://github.com/DataDog/dd-sdk-swift-testing`][1].
 
+    ![](swift_images/swift_package.png)
```

Comment:
This doesn't seem to work @nachoBonafonte ?

Reply:
It's standard markdown, I thought it would be supported

Reply:
Finally fixed

Reply:
thanks